app/metrics.yaml

```diff
@@ -841,6 +841,28 @@ sync_auth:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-09-01"
+  use_email:
+    type: event
+    description: >
+      A user chose to use their email to sign in instead of scanning a QR code, counterpart to "scan_pairing"
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/9834
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/9835#issuecomment-612951889
```

Comment:
This should be a link to the data steward's response, not the review form itself.

Reply:
Ideally, but sometimes we also just link to the PR with both of the forms 🤷‍♀️

Reply:
I was just following the best example I had https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300

Reply:
Every time I've had a review from from glean folks, they've been pretty insistent on it :)